assets/js/pages/Event.tsx

```diff
@@ -100,10 +103,28 @@ const Event = () => {
           </Button>
         </Box>
         <Spacer />
-        <ParticipantsList participants={participants} isModerator={eventInfo.is_moderator} />
+        <ParticipantsList
+          yourName={eventInfo.username}
+          eventChannel={eventChannel}
+          participants={participants}
+          isModerator={eventInfo.is_moderator}
```

Comment:
I think it would better look if we would stay in camelCase, probably you should map values after receiving value.